components/dropdown/dropdown-content-mixin.js

```diff
@@ -973,13 +981,13 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		return false;
 	}
 
-	_getPosition(spaceAround, centerDelta) {
-
+	_getPosition(spaceAround, targetWidth, contentWidth) {
+		const centerDelta = contentWidth - targetWidth;
 		const contentXAdjustment = centerDelta / 2;
-		if (centerDelta <= 0) {
+		if (!this.align && centerDelta <= 0) {
 			return contentXAdjustment * -1;
 		}
-		if (spaceAround.left > contentXAdjustment && spaceAround.right > contentXAdjustment) {
+		if (!this.align && spaceAround.left > contentXAdjustment && spaceAround.right > contentXAdjustment) {
```

Comment:
If align has a value and wider opener then `_getPosition` ends up in here which is not the result we want.